### DISCUSSION TITLE
Why not merge ecoCode-common into ecoCode ?

### DISCUSSION THREAD

**Discussion by standousset** [green-code-initiative/creedengo-common#53] [Polls]
While browsing through the two repos : (1) [ecoCode ](https://github.com/green-code-initiative/ecoCode.git)and (2) [ecoCode-common](https://github.com/green-code-initiative/ecoCode-common), I was wondering why we have those two repos.

The main repo [ecoCode ](https://github.com/green-code-initiative/ecoCode.git) seems like a good place for :
- having the lists of rules (with the matrix of the state of each rule by language)
- AND having the documentation for starting contributions, code of conduct, explanation about tools...

**What are the reasons for the separations ? Too many repos can create confusion.**

cc. : @dedece35 / @glalloue 

_P.S. : I also use the opportunity to (re)test the Discussion feature in GitHub as decided recently on the Slack of the team. If necessary this disussion can be converted into an issue (for instance if we need to merge the two repos) and we can follow the tasks with GitHub issues kanban boards._

**Comment by dedece35** [comment]
Hi @standousset,
"ecocode" project has been renamed to "creedengo".
Thus, old "ecocode" repository has been renamed to "creedengo-rules-specifications" because it is a Java component used in all other plugins.
For me, it seems quite confuse to put the documentation inside this repository.

On the other hand, "creedengo-common" contains also some shell scripts.
